tests/stress_tests/mutex_stress/src/main.c

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <ztest.h>
+
+#define THREAD_COUNT 2
+#define THREAD_STACK_SIZE 1024
+
+struct k_thread thread_array[THREAD_COUNT];
+K_THREAD_STACK_ARRAY_DEFINE(thread_stack, THREAD_COUNT, THREAD_STACK_SIZE);
+
+struct k_thread sem_thread;
+K_THREAD_STACK_DEFINE(sem_thread_stack, THREAD_STACK_SIZE);
+
+struct k_mutex mutex;
+struct k_sem sem;
+struct k_timer timer;
```

Comment:
Make all the global variable static as possible, if they are not used in other place. Just a suggestion.

Reply:
fixed.